packages/runtime/src/interpreter/interpreter.ts

```diff
@@ -29,6 +29,10 @@ export class Interpreter {
   instructions: Operator[];
   instructionIndex: number;
   runtime: Runtime;
+  // The call stack is separate from the data stack. Only callsub and retsub manipulate it.
+  // It is used to provide sub routine functionality
+  callStack: Stack<number>;
+  labelMap: Map<string, number>; // label string mapped to their respective indexes in instructions array
```

Comment:
is "respective indexes" same as `line no.` in TEAL file?

Reply:
If that's the case in instructions array, This array is used for internal mapping(used to jump to specific label instruction)